src/osal/lv_freertos.c

```diff
@@ -193,17 +188,28 @@ lv_result_t lv_thread_sync_wait(lv_thread_sync_t * pxCond)
 {
     lv_result_t lvRes = LV_RESULT_OK;
 
+    /* If the cond is uninitialized, perform initialization. */
+    prvCheckCondInit(pxCond);
```

Comment:
If wait gets called prior of any init, this call will do the initialization.

Reply:
Correct. I applied your feedback after your last review to make it work that way. Is there something you want me to change?